plugins/scaffolder-react/src/next/components/Stepper/guards.ts

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
Wonder if this should be `utils` not `guards`. Even though there is a `typeguard` in it, it feels a little unfortunate to have `hasErrors` in here too. `utils` kind of captures it all, even at risk of it just becoming a kitchen sink.

Reply:
I agree, I'm not actually exporting the guard so I've `utils` it is.